acceptance_tests/config/mapserver/AZ_az-0.9.map

```diff
@@ -4,7 +4,7 @@ MAP
     EXTENT -180 -90 180 90
     UNITS METERS
 
-    # RESOLUTION and DEFRESOLUTION default to 96. If you
+    # RESOLUTION and DEFRESOLUTION default is 96. If you
```

Comment:
Same comment as in https://github.com/camptocamp/c2cgeoportal/pull/9094#discussion_r778589978

here, the "default to" is correct, because of the plural and therefore "default" can be read as a verb
So, either keep as before
RESOLUTION and DEFRESOLUTION default to 96
or
RESOLUTION and DEFRESOLUTION have as default value 96
or
For RESOLUTION and DEFRESOLUTION, the default is 96